packages/framework-types/src/instrumentation/trace-types.ts

```diff
@@ -0,0 +1,55 @@
+import { UUID } from '../concepts'
+import { BoosterConfig } from '../config'
+
+export enum TraceTypes {
+  START,
+  END,
+}
+
+export enum TraceActionTypes {
+  CUSTOM,
```

Comment:
Suggestion: It would be great if the user could use any string in the `@Trace` decorator to be able to define their own actions. This way, they can enable the trace for specific methods of their project. Right now, you only have the ability to enable or disable traces for internal actions. In your project it is all or nothing.